backend/open_webui/utils/chat.py

```
@@ -7,14 +7,17 @@
 import json
 import inspect
+import uuid
+import asyncio
 
-from fastapi import Request
-from starlette.responses import Response, StreamingResponse
+from fastapi import Request, status
+from starlette.responses import Response, StreamingResponse, JSONResponse
 
 
 from open_webui.models.users import UserModel
```

Ruff (0.8.2)

`open_webui.models.users.UserModel` imported but unused

Remove unused import: `open_webui.models.users.UserModel`

(F401)